src/TestSuite/IntegrationTestTrait.php

```diff
@@ -237,11 +244,13 @@ public function setUnlockedFields(array $unlockedFields = []): void
      * Both the POST data and cookie will be populated when this option
      * is enabled. The default parameter names will be used.
      *
+     * @param string $cookieName The name of the csrf token cookie.
      * @return void
      */
-    public function enableCsrfToken(): void
+    public function enableCsrfToken(string $cookieName = 'csrfToken'): void
```

Comment:
Given the change in method signature this needs to target `4.next` branch.

Reply:
This has been updated to 4.next.